src/esportsbot/cogs/MusicCog.py

```diff
@@ -221,119 +211,132 @@ async def removesong(self, ctx: Context, song_index=None):
-        :return: None
+        :return: A boolean if the bot was removed from the voice channel successfully.
         """
+        await ctx.message.delete()
 
         if not self.__check_valid_user_vc(ctx):
             # Checks if the user is in a valid voice channel
-            return
+            return False
 
-        await self.__remove_active_channel(ctx.guild.id)
-        await ctx.message.delete()
-        message = Embed(title="I have left the Voice Channel", colour=EmbedColours.music)
-        await send_timed_message(ctx.channel, embed=message, timer=20)
+        if await self.__remove_active_channel(ctx.guild.id):
+            message = Embed(title="I have left the Voice Channel", colour=EmbedColours.music)
+            await send_timed_message(ctx.channel, embed=message, timer=20)
+            return True
+        return False
 
     @commands.command()
-    async def skipsong(self, ctx: Context):
+    async def skipsong(self, ctx: Context) -> bool:
         """
         Skips the current song. If there are no more songs in the queue, the bot will leave.
         :param ctx: The context of the message.
         :return: None
         """
 
+        await ctx.message.delete()
+
         if not self.__check_valid_user_vc(ctx):
             # Checks if the user is in a valid voice channel
-            return
+            return False
 
         # TODO: Decide if the bot should leave the vc if the queue is empty after skipping.
         if len(self._currently_active.get(ctx.guild.id).get('queue')) == 1:
             # Skipping when only one song in the queue will just kick the bot
             await self.__remove_active_channel(ctx.guild.id)
-            await ctx.message.delete()
-            return
+            return True
 
         await self.__check_next_song(ctx.guild.id)
-        await ctx.message.delete()
         message = Embed(title="Song Skipped!", colour=EmbedColours.music, time=5)
         await send_timed_message(ctx.channel, embed=message)
+        return True
 
     @commands.command()
-    async def listqueue(self, ctx: Context):
+    async def listqueue(self, ctx: Context) -> bool:
         """
         Sends a message of the current queue to the channel the message was sent from.
         :param ctx:The context of the message.
-        :return: None
+        :return: A boolean if sending the message was successful.
         """
 
         # We don't want the song channel to be filled with the queue as it already shows it
         music_channel_in_db = self.__db_accessor.get('music_channels', params={'guild_id': ctx.guild.id})
         if ctx.message.channel.id == music_channel_in_db[0].get('channel_id'):
             # Message is in the songs channel
-            return
+            return False
 
         queue_string = self.__make_queue_list(ctx.guild.id)
 
-        await ctx.channel.send(queue_string)
+        if await ctx.channel.send(queue_string) is not None:
```

Comment:
I've never seen `channel.send` return `None`, instead it throws an exception
One of `discord.HttpException` or  `discord.Forbidden`
or when interacting with stuff that already exists, `discord.NotFound`

Edit: send also throws `InvalidArgument`